packages/db/src/migrations/20200311015053_create_users_table.js

```diff
@@ -0,0 +1,16 @@
+const TABLE_NAME = 'users'
+
+exports.up = knex => {
+  return knex.schema
+    .createTable(TABLE_NAME, table => {
+      table.increments('id').unsigned().primary()
```

Comment:
sequential id's are easy but not considered a best practice. Some third party could derive a lot from it which is not desired. consider using some UUID or hash id's on API side.